apps/api/src/modules/auth/auth.service.ts

```diff
@@ -28,11 +32,29 @@ export class AuthService {
     return null;
   }
 
-  async login(loginUserInput): Promise<LoginResponse> {
+  async login(loginUserInput: LoginUserInput): Promise<LoginResponse> {
     const token = this.configService.getOrThrow<string>('SERVER_API_KEY');
+    const tokenList = await this.setTokenList(loginUserInput.username);
+
     return {
       token,
       user: loginUserInput.username,
+      allKeys: tokenList,
     };
   }
+
+  async setTokenList(inputUserName: string): Promise<string[] | null> {
+    let listOfKeys = null;
+
+    // Get the details of user
+    const entry = await this.usersService.findByUsername(inputUserName);
+
+    // Get all active keys if ADMIN
+    if (entry.userRole === UserRoles.ADMIN) {
+      const allKeyEntries = await this.serverApiKeysService.findAllWithStatusOne();
+      listOfKeys = allKeyEntries.map((key) => key.apiKey);
+    }
```

Comment:
If we remove `token` above, then need to add an else case for non-admin users to get the server API key as well.

Reply:
Added function for this that fetches data from db instead of relying on .env